core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/JsonCodec.scala

```diff
@@ -343,6 +349,13 @@ object JsonCodec {
     Decoder.decodeString.emap(x => DiskStatus.withNameOption(x).toRight(s"Invalid disk status: $x"))
   implicit val diskTypeDecoder: Decoder[DiskType] =
     Decoder.decodeString.emap(x => DiskType.stringToObject.get(x).toRight(s"Invalid disk type: $x"))
+  implicit val gpuTypeDecoder: Decoder[GpuType] =
```

Comment:
Reminder to update swagger :)

Also maybe we could name it `gpuType` and `gpuCount`? Looks like GCP names it `acceleratorType` and `acceleratorCount`.

Reply:
@jdcanas 's front leo PR will update API doc...I think `gpuType` is better since it's widely known, and I remember reading somewhere that seems like there's another type of accelerator (non gpu), altho I can't find the doc at the moment

Reply:
yeah the `gpu` naming sounds fine.. mainly was thinking `s/numOfGpus/gpuCount`

Reply:
still think `gpuCount` sounds cleaner :)

Reply:
just saw this...I think `numOfGpus` is a bit more consistent with existing code, since we have `numOfxxx` and `numberOfxx` in most places

Reply:
ok, fair enough 👍